bot/exts/holidays/valentines/be_my_valentine.py

```diff
@@ -41,21 +41,39 @@ async def lovefest_role(self, ctx: commands.Context) -> None:
         """
         raise MovedCommandError(MOVED_COMMAND)
 
-    @commands.cooldown(1, 1800, commands.BucketType.user)
+    # @commands.cooldown(1, 1800, commands.BucketType.user)
     @commands.group(name="bemyvalentine", invoke_without_command=True)
     async def send_valentine(
-        self, ctx: commands.Context, user: discord.Member, *, valentine_type: str | None = None
+        self, ctx: commands.Context, user: discord.Member, privacy_type: str | None = None, anon: str | None = None, valentine_type: str | None = None
```

Comment:
`typying.Literal` can be used to make discord.py check if the argument is valid https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html#typing-literal

This would simplify the logic later on.